src/openrct2/actions/MazeSetTrackAction.hpp

```diff
@@ -77,11 +77,7 @@ struct MazeSetTrackAction : public GameActionBase<GAME_COMMAND_SET_MAZE_TRACK, G
 
     uint16_t GetActionFlags() const override
     {
-        uint16_t flags = GameAction::GetActionFlags();
-        if (GetFlags() & GAME_COMMAND_FLAG_ALLOW_DURING_PAUSED)
-            flags |= GA_FLAGS::ALLOW_WHILE_PAUSED;
-
-        return flags;
+        return GameAction::GetActionFlags();
```

Comment:
Since this just calls the member function of its parents class, can't this be removed entirely?

Reply:
Thats actually a good point.